charts/pravega-operator/templates/operator.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pravega-operator
```

Comment:
I think we should parameterize the metadata as is typical of charts.   `{{ReleaseName}}`  type stuff.